examples/phewas_scatter.html

```diff
@@ -171,14 +171,16 @@ <h3 style="color: #777">API (for developers)</h3>
       </div>
 
       <script type="text/javascript">
+        var variantForPlot = "10:114758349_C/T";
 
-        // What variant are we using to display a PheWAS?
+        // Throughout this demo, we will match variants of the format 10:100_C/T
+        var VARIANT_PATTERN = /(\d+):(\d+)_([ATGC])\/([ATGC])/;
 
-        var variantChrom = 10;
-        var variantPosition = 114758349;
-        var variantRefAlt = "C/T";
-        // Variants are expressed in format `10:114758349_C/T`;
-        var variant = [variantChrom, ":", variantPosition, "_", variantRefAlt].join("");
+        // Break the variant into constituent parts for setting plot state
+        var variantGroups = VARIANT_PATTERN.exec(variantForPlot);
+        var variantChrom = variantGroups[1];
+        var variantPosition = variantGroups[2];
```

Comment:
When the page first loads it's not quite lined up on the right spot in the gene track, I think b/c variantPosition now ends up as a string at first. This seems to fix it: 

```
var variantPosition = parseInt(variantGroups[2]); 
```

Reply:
Update made; my fault entirely for not refreshing properly.

I'm going to squash the commit history tonight and consolidate what we will send out, and will do some basic regression testing first half of tomorrow with fresher eyes.